languages/python2/bin/build

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "$PYTHON_INDEX_URL" ]]; then
```

Comment:
Should there be an ARG in the docker file for this? Not totally sure how this would be set in the build pipeline

Reply:
Great point! Added JIRA item to track setting environment variables for mirrors (and http proxy...? that might be a separate work item)